book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,286 @@
+If your event needs to start early in the morning, consider hosting a welcome session that is able to be missed if necessary. 
+
+Additionally, consider planning your event around important holidays and cultural celebrations, as well as considering how weekend events will impact different people's ability to attend. 
+Events that occur on weekends or during working hours can negatively impact the ability of people with lived experience who have jobs or caregiving commitments to attend events. 
+By considering when your attendees will be required elsewhere, you will be more likely to find a time that suits everyone. 
+
+(cl-inclusive-events-session-duration)=
+### Session Duration
+Your schedule should take into account breaks and meal times, and carefully plan the length of its sessions. 
+
+If your event takes place in person, make sure to allow for sufficient time for people with lived experience and mobility difficulties to move between locations. 
+For example, consider how long it would take to get to an accessible elevator and then travel between floors, or how far away accessible restrooms are. 
+If your main room has only one exit and a large audience, it may take significantly longer to exit the room and cause attendees stress as they have to rush between sessions. 
+
+Additionally, many people who experience cognitive or physical fatigue find it difficult to actively engage in events that last longer than 40 minutes without a meaningful break. 
+
+This timing can vary depending on individuals, however, both ADHD advocates and people with a lived experience of cognitive impairments (such as stroke, traumatic brain injury, and brain fog) have advocated for 30-40 minute session blocks for activities that require sustained attention and engagement. 
+This timing can be highly dependant on the type of content and the specific audience you are engaging.
+
+As a rule of thumb, try to plan for the most important parts of your event's content to be shared within blocks of time that are under 40 minutes.
+
+Because many events are planned in 1-hour or 1.5-hour timing blocks, consider choosing shorter sessions or incorporating breaks within your session. 
+This can be done in addition to allocating 5-10 minutes for an introduction and session set-up and will help reduce the cognitive strain that many people with lived experience of cognitive impairments are affected by. 
+
+> **Example 1: 1-hour Focus Group Event**
+    - 10 minute welcome, Code of Conduct, and Session introduction
+    - 40 minute Discussion between participants
+    - 10 minute break or 10 minute thank you and session close
+
+> **Example 2: 1-hour Workshop event**
+    - 5 minute introduction 
+    - 20 minute session
+    - 5 minute break
+    - 20 minute session
+    - 10 minute break or 10 minute thank you and session close
+
+As mentioned above, it is best to get feedback and input from a representative with lived experience when planning your session timings. 
```

Comment:
I don't think so (unless you've found it since and then I stand corrected 🙈)

Do you think there should be an explicit suggestion (i.e. a new section) about how to engage and what engagement can mean. 

Right now there are just sentences and examples littered throughout. 

What do you think?